controller/ProviderAdmin.php

```diff
@@ -31,6 +34,44 @@
  */
 class ProviderAdmin extends tao_actions_SaSModule
 {
+    private const FIELDS_EXCLUDED_FROM_LTI_1P3 = [
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolIdentifier',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolPublicKey',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#ToolJwksUrl',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolLaunchUrl',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolOidcLoginInitiationUrl',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolDeploymentIds',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolAudience',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolClientId',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolName',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#toolIdentifier',
+        'http://www.tao.lu/Ontologies/TAOLTI.rdf#ltiVersion',
+    ];
+
+    public function addInstanceForm()
```

Comment:
Please, do not replicate data or overload controller with more responsibilities. Controller is an intermediate. It just forward things, does not deal with things. 

You must extend tao-core as @boajer did (https://github.com/oat-sa/tao-core/pull/2683) in order to filter out some fields based on feature flag.

We are replicating lots of values, constants, etc. This is not good.

Notice also that you will benefit from this PRs: 

- https://github.com/oat-sa/extension-tao-lti/pull/274
- https://github.com/oat-sa/tao-core/pull/2683

So lets try to approve then first.